suppression-backup-service/api/api.go

```diff
@@ -29,16 +28,18 @@ func NewAPI(logger logger.Logger, fullBackup, latestBackup model.File) *API {
 }
 
 func (api *API) Handler(ctx context.Context) http.Handler {
-	srvMux := mux.NewRouter()
-	srvMux.Use(gorillaware.StatMiddleware(ctx, srvMux, stats.Default, "suppression_backup_service"))
+	srvMux := chi.NewMux()
+	srvMux.Use(chiware.StatMiddleware(ctx, srvMux, stats.Default, "suppression_backup_service"))
+	srvMux.Use(middleware.Compress(1))
```

Comment:
please use a proper constant for the compression level
```suggestion
	srvMux.Use(middleware.Compress(gzip.BestSpeed))
```